specification/machinelearningservices/AzureAI.Evaluation/routes.tsp

```diff
@@ -73,4 +73,55 @@ interface Evaluations {
     },
     OkResponse<Evaluation>
   >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Waiting for bug fix: https://github.com/Azure/typespec-azure-pr/issues/3739."
+  @doc("Creates an evaluation schedule.")
+  @route("/evaluations/schedules/create")
+  @post
+  create is Azure.Core.Foundations.Operation<
+    {
+      @doc("Properties of Evaluation Schedule.")
+      @body
+      body: EvaluationSchedule;
+    },
+    ResourceCreatedResponse<EvaluationSchedule>
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Waiting for bug fix: https://github.com/Azure/typespec-azure-pr/issues/3739."
+  @doc("Get an evaluation schedule along with runs.")
+  @route("/evaluations/schedules/{id}")
+  @get
+  get is Azure.Core.Foundations.Operation<
+    {
+      @doc("Identifier of the evaluation schedule.")
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_]*$")
+      @maxLength(254)
+      @path
+      id: string;
+
+    },
+    EvaluationSchedule
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Waiting for bug fix: https://github.com/Azure/typespec-azure-pr/issues/3739."
+  @doc("List evaluations under a schedule")
+  @route("/evaluations/schedules/{id}/jobs")

```

Comment:
Can this be covered under evaluations list using `evaluations/list` api with a param for scheduleId ?

Reply:
we will go with "/evaluations/schedules/{id}/runs" for ignite.